app/api/api/views.py

```diff
@@ -357,17 +364,6 @@ def get_queryset(self):
             parent_dataset__project__members=self.request.user.id,
         ).distinct()
 
-    def create(self, request, *args, **kwargs):
```

Comment:
How come this gets deleted?

Reply:
I thought this one is not needed here. Because we had it in the V2. 
But maybe it is better if we have it here, then override it in V2. So putting it back